.github/pipelines/compliance_check.yml

```diff
@@ -22,36 +22,28 @@ pool:
 steps:
 - checkout: self
 
-- task: Bash@3
-  inputs:
-    targetType: inline
-    script: |
-      rm ./src/promptflow/promptflow/_sdk/data/bulkTestDetails.min.js
-      rm ./src/promptflow/promptflow/_sdk/_service/static/TraceViewPage.min.js
-  displayName: 'Remove min.js'
-
 # https://eng.ms/docs/microsoft-security/cloud-ecosystem-security/azure-security/cloudai-security-fundamentals-engineering/security-integration/guardian-wiki/sdl-azdo-extension/PoliCheck-build-task
 - task: PoliCheck@2
   inputs:
     targetType: 'F'
     targetArgument: '$(sourceLocation)'
     optionsPE: '1'
-    optionsUEPATH: '$(sourceLocation)/scripts/compliance-check/user_exclusion.xml'
-    result: '$(sourceLocation)/scripts/compliance-check/result.tsv'
+    optionsUEPATH: '$(sourceLocation)\scripts\compliance-check\user_exclusion.xml'
+    result: '$(sourceLocation)\scripts\compliance-check\result.tsv'
     optionsXCLASS: 'Geopolitical'
 
 - task: PowerShell@2
   inputs:
     targetType: 'filePath'
-    filePath: '$(sourceLocation)/scripts/compliance-check/Check-PolicheckScan.ps1'
+    filePath: '$(sourceLocation)\scripts\compliance-check\Check-PolicheckScan.ps1'
     arguments: >
-      -policheckResult $(sourceLocation)/scripts/compliance-check/result.tsv
+      -policheckResult $(sourceLocation)\scripts\compliance-check\result.tsv
   displayName: 'Check result'
 
 - task: PublishPipelineArtifact@1
   condition: failed()
   inputs:
-    targetPath: '$(sourceLocation)/scripts/compliance-check/result.tsv'
```

Comment:
Why changed this?

Reply:
Just fix the path inside the pipeline like this
![image](https://github.com/microsoft/promptflow/assets/2208599/23892f6d-87b7-45bc-9468-0fad984941c8)